### ISSUE TITLE
Suggestion: Administrators who join should not be allowed to speak until entering their password with /pass

### ISSUE THREAD

**Issue by MasterlazorX** [ClassiCube/MCGalaxy#535]
As a curious being I am, I thought up of an idea of this, and for awhile, the fact this is still a thing really bothers me.

My suggestion I've thought up, defined administrators who join should not be able to speak until entering their password with "/pass".

Leaving it to where they can speak without even verifying presents two problems in my opinion.
1) A "hacker" can take over a server administrator's CC account and can potentially join and the "hacker" can take advantage of this for negative purposes like impersonating, throwing shade at someone/thing.
2) Likely my biggest issue of this so far, so for instance, you're typing the command and the password to get verified and forget the command slash. A good example of this is it's common for me to use a command like "/b" but accidentally type period in the place of it instead. I think this presents a big security issue for when users share that password on different sites (I know it tells you not to set it as your Minecraft password but back to the point.) If they forget or typo the starting command slash, it would end up showing in chat as ".pass example123". Not going in too much detail into what issues would arise but even though it varies from user to user, this obviously presents potential security issues for admins on servers that use the auth feature.

In conclusion, it should be changed to where players can't speak and when they attempt to speak, they will be prompted with the generic /pass message in the same way as it would show when trying to use commands, along with changing the generic /pass message accordingly to accompany "speaking and using commands".

Thanks!
